src/pages/Admin/Charity/Account/Balances.tsx

```diff
@@ -18,24 +17,28 @@ export default function Balances({ type }: { type: AccountType }) {
       {({ total, free, invested, symbol }) => (
         <div className="grid grid-cols-3 gap-6">
           <Amount title="Total value" symbol={symbol}>
-            {humanize(total, 2)}
+            {total}
           </Amount>
           <Amount title="Invested balances" symbol={symbol}>
-            {humanize(invested, 2)}
+            {invested}
           </Amount>
           <Amount title="Free balances" symbol={symbol}>
-            {humanize(free, 2)}
+            {free}
           </Amount>
         </div>
       )}
     </QueryLoader>
   );
 }
 
-function Amount({
-  classes = "",
-  ...props
-}: PropsWithChildren<{ classes?: string; title: string; symbol: string }>) {
+type Props = {
+  classes?: string;
+  title: string;
+  symbol: string;
+  children: number;
```

Comment:
Using `props.children` like this is a bit confusing, would've been better to use the suggested `value: number` (or whatever other name) field.